torch/sparse/__init__.py

```diff
@@ -9,6 +9,7 @@
 
 
 def addmm(mat, mat1, mat2, beta=1, alpha=1):
```

Comment:
u might have to put `beta=1.0, alpha=1.0`

Reply:
added the test for alpha beta when they are floats, seems like we don't need to do it anymore